scripts/views/eventView.js

```diff
@@ -9,12 +9,41 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
+    if (zip && zip.length === 5) {
```

Comment:
as far as i know 'pattern' has spotty cross browser support. I also don't like the look of the default error message in chrome. I do prefer this overall though